src/adapters/MeetingsJSONAdapter.js

```diff
@@ -558,11 +558,13 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
    */
   async switchCamera(ID, cameraID) {
     await this.updateMeeting(ID, async (meeting) => {
-      this.stopStream(meeting.localVideo.stream);
+      if (meeting.localVideo.stream) this.stopStream(meeting.localVideo.stream);
 
       return {
         localVideo: {
-          stream: await this.getStream({video: {deviceId: {exact: cameraID}}}),
+          stream: meeting.localVideo.stream ? await this.getStream({
```

Comment:
I think this is more complicated, because with these changes the user won't be able to see a preview and switch camera if it is already stopped. We need another place to store the preview stream and the preferred device (when camera is stopped).